lib/compat/wordpress-6.2/get-global-styles-and-settings.php

```diff
@@ -157,38 +136,6 @@ function gutenberg_get_global_stylesheet( $types = array() ) {
 	return $stylesheet;
 }
 
-/**
- * Clean the cache used by the `gutenberg_get_global_stylesheet` function.
- */
-function gutenberg_get_global_stylesheet_clean_cache() {
```

Comment:
This was introduced at https://github.com/WordPress/gutenberg/pull/45679 and is slated for the 14.7 release. I've marked this PR for that release as well, so we can remove.